odk1-src/form-logic.rst

```diff
@@ -240,44 +264,71 @@ without answering the question.
 Setting default responses
 ===========================
 
-To provide a default response to a question,
-put the response value in the :th:`default` column.
+To provide a default response to a question, put a value in the :th:`default` column. Defaults are set once when a record is first created from a form definition. Defaults can either be fixed values (:ref:`static defaults <static-defaults>`) or the result of some expression (:ref:`dynamic defaults <dynamic-defaults>`). 
```

Comment:
`Defaults are set when a record is first created from a form definition. Defaults are set once and only once. Defaults can be...`